typhos/display.py

```diff
@@ -224,18 +284,20 @@ def handle_menu(checked):
 
     def generate_context_menu(self):
         """
-        Generates the custom context menu
-
-        Embedded
-        Detailed
-        Engineering
-        -------------
-        Refresh templates
-        -------------
-        Kind filter > Show hinted
-                      ...
-                      Show only hinted
-        Filter by name
+        Generate the custom context menu.
+
+        .. code::
+
+            Embedded
+            Detailed
+            Engineering
+            -------------
+            Refresh templates
+            -------------
+            Kind filter > Show hinted
+                          ...
+                          Show only hinted
+            Filter by name
```

Comment:
I forgot that when I added this option.
```suggestion
            Filter by name
            Hide Empty Panels
```